web/gds-admin-ui/src/pages/app/details/index.js

```diff
@@ -52,7 +52,7 @@ const VaspDetails = () => {
                         </Col>
                         <Col md={6} xl={4} xxl={4}>
                             <Contact data={vasp?.vasp?.contacts} verifiedContact={vasp?.verified_contacts} />
-                            <AuditLog data={vasp?.vasp?.extra} />
+                            <AuditLog data={vasp?.vasp?.audit_log} />
```

Comment:
@elysee15 it looks like the `AuditLog` got merged into main? 

I did try running this locally and I'm not seeing an AuditLog widget between the contact and the certificate details. I am getting audit log responses back from the API. Am I doing something wrong?

Reply:
I changed how I accessed to the `audit.log`. Is it a `extra` property or `vasp` property ?